samples/seat_massager/consumer/src/main.rs

```diff
@@ -160,30 +129,32 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .consumer_authority
         .expect("consumer_authority must be specified in the config file");
 
+    // Setup the asynchrnous channel for AnswerRequest's.
+    let (tx, rx) = mpsc::channel(100);
```

Comment:
From: https://docs.rs/tokio/latest/tokio/sync/mpsc/fn.unbounded_channel.html

"Note that the amount of available system memory is an implicit bound to the channel. Using an unbounded channel has the ability of causing the process to run out of memory. In this case, the process will be aborted."

I think that it would be best to keep this bounded.  Happy to bump the size if you feel that 100 is too low.

Reply:
I also read that the bounded implementation is more performant than the unbounded implementation.